com.exok.Celeste64.yml

```diff
@@ -0,0 +1,46 @@
+id: com.exok.Celeste64
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet8
+build-options:
+  append-path: "/usr/lib/sdk/dotnet8/bin"
+  append-ld-library-path: "/usr/lib/sdk/dotnet8/lib"
+  env:
+    PKG_CONFIG_PATH: "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet8/lib/pkgconfig"
+command: /app/bin/celeste64/bin/Release/net8.0/linux-x64/Celeste64
+finish-args:
+  - --device=all
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --share=ipc
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=fallback-x11
+
+modules:
+  - name: celeste64
+    buildsystem: simple
+    build-commands:
+      - "install-sdk.sh"
+      - "dotnet publish -c Release --source ./nuget-sources Celeste64.csproj"
+      - "cp -r --remove-destination /run/build/celeste64/ /app/bin/"
+      
+      - install -Dm644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - install -Dm644 ${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 icon.png ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.png
```

Comment:
a PNG is NOT scalable.

Reply:
Could you help me with this one? What is the correct path for a PNG icon?